Products/CMFPlone/tests/testCatalogTool.py

```diff
@@ -1414,6 +1415,21 @@ def test_uuid(self):
         self.assertTrue(wrapped.UID)
         self.assertTrue(uuid == wrapped.UID)
 
+    def test_getIcon(self):
+        from Products.CMFPlone.CatalogTool import getIcon
```

Comment:
getIcon here vs get_icon below?

Reply:
yeah, getIcon is a `plone.indexer.delegate.DelegatingIndexerFactory` and by doing `getIcon.callable` I get the actual function to test.
This is the same way it is done above, with `getObjSize`

Reply:
@frapell yep sorry, I overlooked that line